setup.cfg

```diff
@@ -34,3 +34,22 @@ ignore_missing_imports = True
 [flake8]
 max-line-length = 200
 exclude = venv/*,var/*
+
+# E203 - White space before ':', conflicting with black
+# E712 - comparison to False should be 'if cond is False:' or 'if not cond:', doesn't apply to our code
```

Comment:
Why are we ignoring `E712`? It seems important to have.

https://www.flake8rules.com/rules/E712.html

Reply:
This suggestion doesn't work for our code. Changing them could cause errors.

Reply:
I think that's a problem with our code then and it should be fixed.

Reply:
```>>> None is False
False
>>> None == False
False
>>> not None
True
>>> 0 == False
True
>>> 0 is False
False
```
This is the case that the suggestion doesn't work well.